src/nxdoc/getting-started/best-practices/choosing-how-to-import-data-in-the-nuxeo-platform.md

```diff
@@ -169,58 +172,71 @@ May it be a file system binary store, an S3 binary store or an Azure Object stor
+The [Nuxeo Platform Scan Importer](https://connect.nuxeo.com/nuxeo/site/marketplace/package/nuxeo-scan-importer) is a submodule of the importer framework and is typically used for the output of a digitalization chain. Nuxeo Platform Scan Importer listens to a given folder and will import all content referenced via XML files, with their metadata, etc. Scan importer also offers very advanced XML <--> documents mapping possibilities, with ability to use some automation processing during the import phase.
 
 **Customization Language**
 
 Java
 
 **Customization Logic**
 
-The importer framework offers many customization possibilities. You can read the [Nuxeo Platform Importer documentation]({{page page='nuxeo-bulk-document-importer'}}) to learn more.
+Scan Importer is configurable via XML extensions for the metadata mapping. The documentation provides links to [simple and advanced use cases]({{page page='scan-documents-importer'}})
 
-**Limitations**
+#### Nuxeo Stream Importer
 
-Files must be available on a file system mounted on the Nuxeo server.
+**Features**
 
-### Nuxeo Scan Importer
+The [Nuxeo Stream Importer](https://github.com/nuxeo/nuxeo/tree/master/addons/nuxeo-platform-importer/nuxeo-importer-stream) defines a producer/consumer pattern and uses the Log features provided by Nuxeo Stream. The Log is used to perform mass import. It decouples the Extraction/Transformation from the Load (using the ETL terminology). The extraction and transformation is done by a document message producer with custom logic. This module comes with a random document and a random blob generator, that does the same job as the random importer of the `nuxeo-importer-core` module. The load into Nuxeo is done with a generic consumer.
+Automation operations are exposed to run producers and consumers.
 
-**Features**
+Two options are offered to perform the import:
+1. Two steps import: Generate and Import documents with blobs.
+1. Four steps import: Generate and Import blobs, then Generate and Import documents.
 
-The [Nuxeo Platform Scan Importer](https://connect.nuxeo.com/nuxeo/site/marketplace/package/nuxeo-scan-importer) is a submodule of the importer framework and is typically used for the output of a digitalization chain. Nuxeo Platform Scan Importer listens to a given folder and will import all content referenced via XML files, with their metadata, etc. Scan importer also offers very advanced XML <--> documents mapping possibilities, with ability to use some automation processing during the import phase.
+{{#> callout type='info' heading='Recommended mechanism for big volumes'}}
+The Nuxeo Stream Importer is the recommended way of performing document imports in a Nuxeo repository as it brings fault tolerance, durability, immutability and ordering to a distributed system. Therefore, it provides failover: Any Nuxeo nodes can be restarted during the import, the import can be re-executed from the start, etc.
+{{/callout}}
```

Comment:
is the recommended way of performing mass import with a decoupled extraction and load phases, the import is fault-tolerant, any Nuxeo nodes can be restarted during the import, it is also possible to re-execute an import from the beginning.